routers/web/repo/setting/require_action.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package setting
+
+import (
+	"errors"
+	"net/http"
+
+	actions_model "code.gitea.io/gitea/models/actions"
+	"code.gitea.io/gitea/models/db"
+	"code.gitea.io/gitea/modules/base"
+	shared "code.gitea.io/gitea/routers/web/shared/actions"
+	"code.gitea.io/gitea/services/context"
+)
+
+const (
+	tplOrgRequireAction base.TplName = "org/settings/actions"
+)
+
+type requireActionsCtx struct {
+	OrgID                 int64
+	IsOrg                 bool
+	RequireActionTemplate base.TplName
+	RedirectLink          string
+}
+
+func getRequireActionCtx(ctx *context.Context) (*requireActionsCtx, error) {
+	if ctx.Data["PageIsOrgSettings"] == true {
+		return &requireActionsCtx{
+			OrgID:                 ctx.Org.Organization.ID,
+			IsOrg:                 true,
+			RequireActionTemplate: tplOrgRequireAction,
+			RedirectLink:          ctx.Org.OrgLink + "/settings/actions/require_action",
+		}, nil
+	}
+	return nil, errors.New("unable to set Require Actions context")
+}
+
+// Listing all RequireAction
+func RequireAction(ctx *context.Context) {
+	ctx.Data["ActionsTitle"] = ctx.Tr("actions.requires")
```

Comment:
I don't know what `ActionsTitle` is used for. Maybe `ctx.Data["Title"] = ctx.Tr("actions.require_action")` ?